velox/common/caching/ScanTracker.h

```diff
@@ -78,9 +79,20 @@ struct TrackingData {
   int64_t readBytes{};
   int32_t numReferences{};
   int32_t numReads{};
-  void incrementReference(uint64_t bytes) {
+
+  // Marks that 'bytes' worth of data in the tracked object has been
+  // accessed. If 'bytes' is larger than a single access quantum, the
+  // reference counts for as many accesses tat are needed to cover
+  // 'bytes'. When reading a large object, we will get a read per
+  // quantum. So then if the referenced and read quanta match, we know
+  // that the object is densely read.
+  void incrementReference(uint64_t bytes, int32_t quantum = 0) {
```

Comment:
Orri, thank you for a very nice description. Why is quantum an optional parameter with default value of 0. Shouldn't we always specify it? Would it make sense to rename it to loadQuantum for clarity? Also, quantum is constant, it would be more natural to pass it in the constructor, no?